From 312a2d138e88454fa38acb8c2856a9a426fd55eb Mon Sep 17 00:00:00 2001
From: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date: Thu, 30 Jul 2026 02:27:34 +0000
Subject: [PATCH] Update elements/components/audit.bst to v4.2.1-0

Updates elements/components/audit.bst to v4.2.1-0 (0065960)
Previous version was v4.2-0 (29eec1e)
---
 elements/components/audit.bst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/components/audit.bst b/elements/components/audit.bst
index 36602d214..2dbc77dad 100644
--- a/elements/components/audit.bst
+++ b/elements/components/audit.bst
@@ -41,7 +41,7 @@ sources:
   track: v*.*
   exclude:
   - "*b*"
-  ref: v4.2-0-g29eec1e2904f55e88f9eeef483ebc1c9062c394a
+  ref: v4.2.1-0-g00659601cc23dc243bb0f451977636ff77fcf80f
 - kind: local
   path: files/audit/tmpfiles-audit.conf
   directory: freedesktop-sdk
-- 
2.55.0

